### PR TITLE
binary classification on good/bad learner

### PR DESCRIPTION
**Pull request by haoyuan-l**
training accuracy is 91% for 200 epochs.

Working in progress...

To do:
1. Prediction scripts
2. Plotting training curve
3. hyper-parameters tuning
4. move on to the next mode...

### CODE REVIEW

**Comment by haoyuan-l** [comment]
Achieved 92% accuracy on the training set, and 95% accuracy on the testing set

**Comment by haoyuan-l** [comment]
To do:
       (Maybe) Perform Time series prediction on lactation

**Review by TKlingstrom — APPROVED** [review]
Wow, I look forward to looking at this when we get together next week.
